### PR TITLE
Posting feature

### PR DESCRIPTION
**Pull request by danyguajardo**
Overall, the individual pin images now load, the full screen images now have good quality and have captions, the pins are displayed as circles

### CODE REVIEW

File: Fitness App/Models/MapPin.m [hunk 1]
```diff
@@ -19,18 +19,18 @@ @implementation MapPin
...
+////    annotationView.enabled = YES;
+////    annotationView.canShowCallout = YES;
+////    annotationView.image = self.image;
+////    UIButton *infoButton = [UIButton buttonWithType:UIButtonTypeDetailDisclosure];
+////    annotationView.rightCalloutAccessoryView = infoButton;
+////
+////    [infoButton addTarget:self action:@selector(buttonPressed:) forControlEvents:UIControlEventTouchUpInside];
+////
+////    return annotationView;
+//}
```
Comment (britneyting) [inline Fitness App/Models/MapPin.m:33]: Maybe delete these commented-out lines if you don't need them anymore?

File: Fitness App/View Controllers/FullPostViewController.m [hunk 2]
```diff
@@ -18,14 +18,16 @@ @implementation FullPostViewController
 
 - (void)viewDidLoad {
     [super viewDidLoad];
-    // Do any additional setup after loading the view.
     self.scrollView.delegate = self;
     self.imageView.image = self.photo;
+    self.caption.text = self.annotation.subtitletitle;
+    NSLog (@"subtitle third time %@", self.caption.text);
+
+//    self.caption.text = caption.text;
```
Comment (britneyting) [inline Fitness App/View Controllers/FullPostViewController.m:26]: delete commented out code?

**Review by britneyting — APPROVED** [review]
yay, it all looks good :D

File: Fitness App/Models/MapPin.m (hunk 1 above)
Comment (danyguajardo) [reply Fitness App/Models/MapPin.m:33]: nice point, actually I don't use anything in that file any more so I will delete the entire thing!

**Pull request closed without merging**